site/public/mjs/activity-dashboard.js

```diff
@@ -69,8 +69,14 @@ export function comparator (row1, row2, n, dir) {
 }
 
 // if n == 0 or n == 8
-// returns true if x < y, digits < strings < empty strings
+// returns true if x < y, empty strings < digits < strings
 export function helper (x, y, i) {
+    if (x != '' && y == '') {
+        return false;
+    }
+    else if (x == '' && y != '') {
```

Comment:
It would be better to use the strict (`===` and `!==`) operators here instead of `==` and `!=`